hudi-hadoop-common/src/test/java/org/apache/hudi/io/hadoop/TestHoodieHBaseHFileReaderWriter.java

```diff
@@ -63,7 +63,8 @@
 import static org.apache.hudi.io.storage.TestHoodieReaderWriterUtils.writeHFileForTesting;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class TestHoodieHBaseHFileReaderWriter extends TestHoodieHFileReaderWriterBase {
+@Disabled("HUDI-9084")
```

Comment:
Are the tests still broken or we can re-enable the tests now after your recent fixes?